packages/arb-avm-cpp/avm_values/src/value.cpp

```diff
@@ -25,6 +25,152 @@
 #include <iomanip>
 #include <ostream>
 
+Value::Value() : Value(Tuple()) {}
+Value::Value(Tuple tup) : Value(0) {
+    inner.tagged.tag = value_tuple_tag;
+    inner.tagged.inner.tuple = std::move(tup);
+}
+Value::Value(uint64_t num) : Value(uint256_t(num)) {}
+Value::Value(uint256_t num)
```

Comment:
Might be more efficient to pass by reference here thought maybe not since `uint256_t` is pretty small